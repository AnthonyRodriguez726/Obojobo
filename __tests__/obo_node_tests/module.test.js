const fs = require('fs')
const xmlToDraftObject = require('../../xml-to-draft-object')

it('Correctly converts xml to Module node', () => {
  const xml = fs.readFileSync('./__tests__/obo_node_xml/module.xml', 'utf8')
  expect(xmlToDraftObject(xml)).toMatchSnapshot()
})