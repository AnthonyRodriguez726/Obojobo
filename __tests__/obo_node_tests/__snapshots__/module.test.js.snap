// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly converts xml to Module node 1`] = `
Object {
  "children": Array [
    Object {
      "children": Array [],
      "content": Object {},
      "id": null,
      "type": "ObojoboDraft.Sections.Content",
    },
  ],
  "content": Object {
    "title": "test",
  },
  "id": null,
  "type": "ObojoboDraft.Modules.Module",
}
`;
